flow/project.py

```diff
@@ -2283,11 +2458,11 @@ def _execute_operation(self, operation, timeout=None, pretend=False):
                 "Executing operation '{}' with current interpreter "
                 "process ({}).".format(operation, os.getpid()))
             try:
-                self._operations[operation.name](operation.job)
+                self._operations[operation.name](*operation._jobs)
             except Exception as e:
                 raise UserOperationError(
                     'An exception was raised during operation {operation.name} '
-                    'for job {operation.job}.'.format(operation=operation)) from e
+                    'for job {operation._jobs[0]}.'.format(operation=operation)) from e
```

Comment:
You don't need to in this PR in my opinion, but this is another example of us needing to change `job` to `jobs` in a future PR.

Reply:
I have inserted a command `assert len(operation._jobs) == 1` above and throughout the codebase.
This was suggested by @csadorf so we know that what things we have to change when we include actual aggregation.

Reply:
That only works with testing as well. We won't raise the assert until we actually test aggregation, but that does help. This wasn't really an issue just an example.